Frameworks/UIKit/UIColor.mm

```diff
@@ -33,790 +33,479 @@
+    return color;
 }
 
 /**
- @Status Interoperable
+@Status Interoperable
 */
-+ (UIColor*)groupTableViewBackgroundColor {
-    return [self colorWithRed:197.f / 255.f green:204.f / 255.f blue:210.f / 255.f alpha:1.0f];
++ (UIColor*)brownColor {
+    static UIColor* color = [UIColor colorWithRed:0.65f green:0.35f blue:0.0f alpha:1.0f];
+    return color;
 }
 
 /**
- @Status Stub
+@Status Interoperable
 */
-+ (UIColor*)viewFlipsideBackgroundColor {
-    return [self whiteColor];
++ (UIColor*)clearColor {
+    static UIColor* color = [UIColor colorWithWhite:0.0f alpha:0.0f];
+    return color;
 }
 
 /**
- @Status Stub
+@Status Interoperable
 */
-+ (UIColor*)windowsControlFocusedColor {
-    return [self colorWithRed:0.19f green:0.46f blue:0.73f alpha:1.0f];
++ (UIColor*)cyanColor {
+    static UIColor* color = [UIColor colorWithRed:0.0f green:1.0f blue:1.0f alpha:1.0f];
+    return color;
 }
 
 /**
- @Status Stub
+@Status Interoperable
 */
-+ (UIColor*)windowsControlDefaultBackgroundColor {
-    return [self colorWithRed:0.95f green:0.95f blue:0.95f alpha:1.0f];
++ (UIColor*)darkGrayColor {
+    static UIColor* color = [UIColor colorWithRed:0.25f green:0.25f blue:0.25f alpha:1.0f];
+    return color;
 }
 
 /**
- @Status Stub
+@Status Interoperable
 */
-+ (UIColor*)_windowsTableViewCellSelectionBackgroundColor {
-    return [self colorWithRed:0.63 green:0.79 blue:0.89 alpha:1.0];
++ (UIColor*)greenColor {
+    static UIColor* color = [UIColor colorWithRed:0.0f green:1.0f blue:0.0f alpha:1.0f];
+    return color;
 }
 
 /**
- @Status Stub
+@Status Interoperable
 */
-+ (UIColor*)windowsTableViewSelectionBackgroundColor {
-    UNIMPLEMENTED();
-    return StubReturn();
++ (UIColor*)lightGrayColor {
+    static UIColor* color = [UIColor colorWithRed:0.75f green:0.75f blue:0.75f alpha:1.0f];
+    return color;
 }
 
 /**
- @Status Interoperable
+@Status Interoperable
 */
-- (BOOL)getHue:(float*)h saturation:(float*)s brightness:(float*)v alpha:(float*)a {
-    hsv out;
-    rgb in;
-
-    in.r = _components.r;
-    in.b = _components.b;
-    in.g = _components.g;
-
-    out = rgb2hsv(in);
-
-    if (h) {
-        *h = (float)(out.h / 360.0);
-    }
-    if (s) {
-        *s = (float)out.s;
-    }
-    if (v) {
-        *v = (float)out.v;
-    }
-    if (a) {
-        *a = _components.a;
-    }
-
-    return YES;
++ (UIColor*)magentaColor {
+    static UIColor* color = [UIColor colorWithRed:1.0f green:0.0f blue:1.0f alpha:1.0f];
+    return color;
 }
 
 /**
- @Status Interoperable
+@Status Interoperable
 */
-- (BOOL)getRed:(float*)r green:(float*)g blue:(float*)b alpha:(float*)a {
-    if (r) {
-        *r = _components.r;
-    }
-    if (g) {
-        *g = _components.g;
-    }
-    if (b) {
-        *b = _components.b;
-    }
-    if (a) {
-        *a = _components.a;
-    }
-
-    return YES;
++ (UIColor*)orangeColor {
+    static UIColor* color = [UIColor colorWithRed:1.0f green:0.5f blue:0.0f alpha:1.0f];
```

Comment:
nit: could these be stuck in a StrongId so they get unloaded w/ the module... or something like that